Nodejs/Product/Nodejs/SharedProject/ProjectNode.CopyPaste.cs

```diff
@@ -1113,7 +1113,6 @@ private Addition CanAddFileFromProjectReference(string projectRef, string target
                         {
                             throw;
                         }
-                        TaskDialog.ForException(this.Project.Site, e, string.Empty, this.Project.IssueTrackerUrl).ShowModal();
```

Comment:
No message box for this one?

Reply:
This only pointed the user to the a new issue page... which, without a stack or a repo project is pointless.